draft-ietf-wimse-s2s-protocol.md

```diff
@@ -314,6 +317,10 @@ Note that per {{RFC9110}}, header field names are case insensitive;
 thus, `Workload-Identity-Token`, `workload-identity-token`, `WORKLOAD-IDENTITY-TOKEN`,
 etc., are all valid and equivalent header field names. However, case is significant in the header field value.
 
+### A note on `iss` claim and key distribution {#wit-iss-note}
+
+It is RECOMMENDED that the WIT carries an `iss` claim. This specification itself does not make use of a potential `iss` claim but also carries the {{trust-domain}} in the {{workload-identifier}}. Implementations MAY include the `iss` claim in the form of a `https`-URL to facilitate key distribution via {{OpenID.Discovery}} or {{!RFC8414}} but alternative key distributions may make use of the {{trust-domain}} of the {{workload-identifier}} carried in the required `sub` claim.
```

Comment:
```suggestion
It is RECOMMENDED that the WIT carries an `iss` claim. This specification itself does not make use of a potential `iss` claim but also carries the trust domain in the workload identifier ({{workload-identifier}}). Implementations MAY include the `iss` claim in the form of a `https` URL to facilitate key distribution via {{OpenID.Discovery}} or {{!RFC8414}} but alternative key distribution methods may make use of the trust domain included in the workload identifier which is carried in the mandatory `sub` claim.
```

Reply:
@arndt-s note that {{trust-domain}} translates in "section XXX" - it's a reference.

Reply:
Thank you!